cmake/MurMurHashConfig.cmake

```diff
@@ -0,0 +1,33 @@
+#.rst:
+# FindMurMurHash
+# -------
+#
+# Find libMurMurhash, An implementation of the MurMurHash3 function in C++. 
+# MurmurHash3 was originally written by Austin Appleby and has been released into the public domain.
+#
+# Result variables
```

Comment:
For this and the other LROSS pull requests: 

I do not think the <repo name>Config.cmake are necessary. Further, they are hard to test also so I believe all of the special cmake config files should no be installed and in fact removed
(Murmurhash, FileIO, StopWatch, DeathKnell, Queuenado)